packages/core/src/context.ts

```diff
@@ -176,6 +177,7 @@ export function defaultContext(): Context {
     luckysheetfile: [],
     defaultcolumnNum: 60,
     defaultrowNum: 84,
+	adddefaultrows: 50,
```

Comment:
Lint errors, try running ```yarn lint --fix``` to see if it works.

Reply:
Fixed Lint issue and Change the variable as lowerCamelCase. @shuanhukexuejia. Please review.